src/components/shared/Camera.tsx

```diff
@@ -80,8 +81,22 @@ export class Camera extends React.Component<CameraProps, CameraState> {
       maxDuration: 10 // seconds
     });
 
+    var uri = recordResponse.uri;
+    try {
+      const cropOptions = {
+        cropWidth: 480,
+        cropHeight: 480,
+        cropOffsetX: 0,
+        cropOffsetY: 80 // TODO: Hack - chose a small offset that looked good. Make preview show exactly what's recorded to remove
```

Comment:
Would this be helpful for that? https://github.com/react-native-community/react-native-camera/pull/1636

Also should we change https://github.com/rahafoundation/raha-mobile-app/blob/4e559cf51e7d9cf0df851fc09f83f8cc91cdd796/src/components/shared/Camera.tsx#L79 ?